src/universal-dapp.js

```diff
@@ -53,30 +53,6 @@ UniversalDApp.prototype.getAccounts = function (cb) {
     }
 };
 
-UniversalDApp.prototype.getBalance = function (address, cb) {
```

Comment:
This was specifically added for a new feature - to display account balances after each transaction.